homework_weekend/js/app.js

```diff
@@ -0,0 +1,85 @@
+// question 1
+
+
+// parametes are the names listed in the function definitinon and agruments are the real
+//reall values passed to the function
+
+// b. a console prints out the function and a return returns the value of the function
+
+// c. Scope determines the accessibility of variables, objects, and functions from 
+//different parts of the code.   
+
+function checkPalindrome(str); 
```

Comment:
no semicolon here—you need curly braces after a function to put the code in like this: 
function checkPalindrome(str) {
    // code goes here
    // code
    // some more code
}